src/Umbraco.Forms.Integrations.Crm.Hubspot/HubspotServerVariablesParsingHandler.cs

```diff
@@ -51,6 +55,12 @@ public void Handle(ServerVariablesParsingNotification notification)
             umbracoUrls["umbracoFormsIntegrationsCrmHubspotBaseUrl"] =
                 _linkGenerator.GetUmbracoApiServiceBaseUrl<HubspotController>(controller => controller.GetAllProperties());
 
+            if (serverVars.ContainsKey("umbracoPlugins"))
```

Comment:
Maybe better here to break up this long single value into an object value for the plugin, even if there's only one property on that object at the moment.  Would be more extensible in future and align with the Forms example I shared.

So something like:

```
        private void AddFormsConfigurationToServerVariables(IDictionary<string, object> dictionary)
        {
            if (dictionary.ContainsKey("umbracoPlugins"))
            {
                var umbracoPlugins = (Dictionary<string, object>)dictionary["umbracoPlugins"];
                umbracoPlugins.Add("umbracoFormsIntegrationsCrmHubspot", new ClientSideConfiguration
                {
                    AllowContactUpdate = _settings.AllowContactUpdate
                });
            }
        }

        [DataContract]
        internal sealed class ClientSideConfiguration
        {
            [DataMember(Name = "allowContactUpdate")]
            public bool AllowContactUpdate { get; set; }
        }
    }
```